src/Sylius/Component/Core/Model/OrderItem.php

```diff
@@ -57,4 +57,45 @@ public function equals(BaseOrderItemInterface $item)
             && $item->getVariant() === $this->variant
         );
     }
+
+    /**
+     * Returns sum of neutral and non neutral tax adjustments on order item and total tax of units.
+     *
+     * {@inheritdoc}
+     */
+    public function getTaxTotal()
+    {
+        $taxTotal = 0;
+
+        foreach ($this->getAdjustments(AdjustmentInterface::TAX_ADJUSTMENT) as $taxAdjustment) {
+            $taxTotal += $taxAdjustment->getAmount();
+        }
+
+        foreach ($this->units as $unit) {
+            $taxTotal += $unit->getTaxTotal();
+        }
+
+        return $taxTotal;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDiscountedUnitPrice()
+    {
+        $orderItemDiscounts = 0;
+        foreach ($this->units as $unit) {
+            $orderItemDiscounts += $unit->getAdjustmentsTotal(AdjustmentInterface::ORDER_ITEM_PROMOTION_ADJUSTMENT);
```

Comment:
I was more thinking of $10 off your order split between 1 `OrderItem` with 3 `OrderItemUnits` but in this case I guess it's an `ORDER_PROMOTION_ADJUSTMENT`.

And you're excluding these because you want only the discounts that were directly associated with this `OrderItem` not those that were derived against it's units as a way of financially associating discount?